pom.xml

```diff
@@ -763,12 +763,23 @@
                 <artifactId>apache-james-mailbox-elasticsearch</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>${james.groupId}</groupId>
+                <artifactId>apache-james-mailbox-elasticsearch-v7</artifactId>
+                <version>${project.version}</version>
+            </dependency>
             <dependency>
                 <groupId>${james.groupId}</groupId>
                 <artifactId>apache-james-mailbox-elasticsearch</artifactId>
                 <version>${project.version}</version>
                 <type>test-jar</type>
             </dependency>
+            <dependency>
+                <groupId>${james.groupId}</groupId>
+                <artifactId>apache-james-mailbox-elasticsearch-v7</artifactId>
+                <version>${project.version}</version>
+                <type>test-jar</type>
+            </dependency>
```

Comment:
Why do we still need the v6 version of some modules here?

Reply:
We do not. I will remove them